code/gradle/distribution.gradle

```diff
@@ -40,13 +40,19 @@ ext {
             include 'docs/**'
         }
     }
-
+    
     programDistsImage = copySpec {
         from("$buildDir/libs") {
             include 'pcgen-*.jar'
             // Use a regular expression to map the file name
             rename "(.+)-$version(.+)", '$1$2'
         }
+        from("$buildDir/launch4j") {
```

Comment:
Had to refactor this image to separate executable files from the scripts in order to accommodate @LegacyKing requests.

That said, all the zip distributions should be verified before merging this.